dialogbox.pro

```diff
@@ -1,6 +1,6 @@
 TEMPLATE = app
 CONFIG += qt thread release
-QT +=
+QT += widgets
```

Comment:
Please consider adding modules conditionally as it works for both versions of qt:
```
greaterThan(4, QT_MAJOR_VERSION): QT += widgets
```